apps/frontend/src/pages/auth/libs/maps/auth-route-to-header.map.ts

```diff
@@ -0,0 +1,13 @@
+import { AppRoute } from "~/libs/enums/app-route.enum.js";
```

Comment:
```suggestion
import { AppRoute } from "~/libs/enums/enums.js";
```

Reply:
If I do Value<typeOf AppRoute>, typescript requires me to provide an entry for every route to the map. Creating RoutesWithHeader type just saves a lot of headache in the future, it is also gives a more descriptive type hint when working with AuthRouteToHeader since it gives the idea that only certain keys yield actual result